src/components/FileManager/FileManager.tsx

```diff
@@ -170,7 +186,8 @@ export const FileManager: React.FC<{
       )
       return (
         designation !== undefined &&
-        designationWithFileSectionsMap.get(designation) === fileSection
+        designationWithFileSectionsMap.get(designation) === fileSection &&
+        !inlineAttachmentsIds.includes(element.id)
```

Comment:
`inlineAttachmentsIds` could be `Set` instead of an array, and using `has` function of the set here will be faster.